src/lib/context-menu.ts

```diff
@@ -307,13 +308,17 @@ export class ContextMenu {
                 const footnotesElement = findChildrenByType(
                   this.node,
                   schema.nodes.footnotes_element
-                )
-                if (!footnotesElement.length) {
+                ).pop()?.node
+                if (
+                  !footnotesElement ||
+                  isDeleted(footnotesElement) ||
+                  isRejectedInsert(footnotesElement)
```

Comment:
when we move back the rejected suggestion will end up having two footnote element, in that case we should change the change state of footnote from rejected to pending. or if it's deleted it should be reverted

Reply:
Can you please elaborate more on your point here?

Reply:
is it ok to have two footnote elements?

Reply:
No, but we are assuming a normal flow without a revert. This is not the only case; we also encounter similar situations in the editor. I recall Panos mentioning in the past that he needs to consider such scenarios.